Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7 AS py-deps
+COPY ./Pipfile ./Pipfile.lock ./
+RUN pip install pipenv && \
+    pipenv install --deploy --system --ignore-pipfile --clear
+
+FROM python:3.7-alpine
```

Comment:
nit: minor concern with alpine/musl vs debian/glibc performance, but I think we can wait to see how this performs